pkg/kubectl/cmd/rollout/rollout_undo.go

```diff
@@ -132,7 +131,7 @@ func (o *UndoOptions) Validate() error {
 
 func (o *UndoOptions) RunUndo() error {
 	r := o.Builder().
-		WithScheme(legacyscheme.Scheme).
+		Unstructured().
```

Comment:
`WithScheme(scheme.Scheme, scheme.Scheme.PrioritizedVersionsAllGroups()...).` should be used here, not unstructured.